cluster/v1alpha1/types.go

```diff
@@ -104,3 +104,40 @@ type ManagedClusterSetBindingList struct {
 	// Items is a list of ManagedClusterSetBinding.
 	Items []ManagedClusterSetBinding `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope="Cluster"
+
+// ClusterClaim represents cluster information that a managed cluster claims
```

Comment:
It is better that we also doc the wellknown name here I think

Reply:
Added.